contracts/protected/ProtectedNFT.sol

```diff
@@ -53,8 +52,8 @@ abstract contract ProtectedNFT is IProtected, Versioned, IERC6454, ERC721, Ownab
   // @param tokenId_ The id of the token.
   modifier onlyProtectorForTokenId(uint256 tokenId_) {
     address owner_ = ownerOf(tokenId_);
```

Comment:
owner_ is defined but not used.

Reply:
Right. I remove the modifier altogether.